rpcs3/Emu/RSX/Core/RSXContext.h

```diff
@@ -3,6 +3,7 @@
 #include <util/types.hpp>
 #include "Emu/Cell/lv2/sys_rsx.h"
 #include "Emu/RSX/GCM.h"
+#include "Emu/RSX/rsx_utils.h"

```

Comment:
What are the header include additions for?

Reply:
I use the clangd extension for VS Code. Sometimes I open a file and it shows errors because includes are missing. So this is specifically for clangd, it does not matter for building.

Reply:
Okay then.

Reply:
All this does it increase the memory footprint during builds...

Reply:
Move to CPP files that require it then.

Reply:
in line 29, there is utils::address_range tile_align(const rsx::address_range& range) const; wwhich is funny and that is why I think it requires rsx_utils.h
use utils::namespace there instead or include add a using statement in this file so it can be dropped.
The other includes seem correct

Reply:
Nah I will merge and fix later.